rust-connectors/sources/mqtt/README.md

```diff
@@ -14,12 +14,16 @@ MQTT V3.1.1 and V5
 
 Controls the Source connector
 
-| Option     | default  | type   | description |
-| :---       | :---     | :---   | :----       |
-| timeout    | 60       | u64    | mqtt broker connect timeout in seconds |
-| mqtt_url   | -        | String | mqtt broker Host |
-| mqtt_topic | -        | String | mqtt topic to subscribe and source events from |
-| client_id  | UUID V4  | String | mqtt client ID |
+| Option        | default  | type   | description                                             |
+| :---          | :---     | :---   | :----                                                   |
+| timeout       | 60       | u64    | mqtt broker connect timeout in seconds                  |
+| mqtt_url      | -        | String | mqtt broker Host                                        |
+| mqtt_topic    | -        | String | mqtt topic to subscribe and source events from          |
+| mqtt_tls      | -        | Flag   | Enables TLS on the MQTT connection                      |
+| mqtt_port     | -        | u16    | Port to use for the MQTT connection                     |
+| mqtt_username | -        | String | mqtt username to authenticate with                      |
+| MQTT_PASSWORD | -        | String | mqtt password to authenticate with in the secrets list  |
```

Comment:
these caps look mean(ish) :eyes: or is it our way of trying to differentiate "secrets" from env?

Reply:
Yeah. I'm open to having them be it's own table.